CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
   - Removed SDX common clone in docker
   - Restore defaults for qids 130, 131, 132.
   - Remove pip `--require-hashes` requirement
+  - Populate default dates on certain surveys from metadata
```

Comment:
Can we be more specific about which surveys this will be changed for? The only survey we will actually get that might contain default dates is RSI

Reply:
Fixed